packages/swapper/src/swappers/thorchain/buildThorTrade/buildThorTrade.ts

```diff
@@ -64,7 +64,7 @@ export const buildTrade = async ({
         gasLimit:
           (quote as TradeQuote<KnownChainIds.EthereumMainnet>).feeData.chainSpecific
             ?.estimatedGas ?? '0',
-        tradeFee: quote.feeData.tradeFee,
+        buyAssetTradeFeeUsd: quote.feeData.tradeFee,
```

Comment:
```suggestion
        buyAssetTradeFeeUsd: quote.feeData.buyAssetTradeFeeUsd,
```

Reply:
Actioned in https://github.com/shapeshift/lib/pull/1038/commits/deb7c95d42e6ce7accc9dd411351f123ea71c814.